App.News/Models/NewsDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace App.News.Models
+{
+    public class NewsDTO
+    {
+        public int id;
+        public string PhotoUrl { get; set; }
```

Comment:
1. Missing empty line between line of code.
2. Class properties must start with a capital letter
3.Missing {get;set;} after int id declaration